problems/4.arrays.js

```diff
@@ -12,15 +16,19 @@ function firstAndLastAverage() {
 * @param {number[]} arr - The input array
 * @returns {number[]} - The duplicated array
 */
-function duplicateArray() {
-
+const duplicateArray = (a, b) => {
+  let a = [1, 2, 3, 4, 5];
+  let b = [1, 2, 3, 4, 5];
+return (a.concat(b));
```

Comment:
functions only takes in 1 array. We don't care what it is, just that it is an array. 
arr.concat(arr)